libs/openchallenges/ui/src/lib/search-dropdown-filter/search-dropdown-filter.component.html

```diff
@@ -8,6 +8,11 @@
   [overlayOptions]="overlayOptions"
   [filter]="filter"
   (onFilter)="onSearch($event)"
+  [virtualScroll]="true"
+  [virtualScrollItemSize]="50"
+  [virtualScrollOptions]="scrollOptions"
+  [lazy]="lazy"
+  (onLazyLoad)="onLazyLoad($event)"
 >
   <!-- custom search bar to avoid conflicts against default filter func -->
   <ng-template *ngIf="filterByApiClient" pTemplate="header">
```

Comment:
In VS Code, `webhint` shows a warning for the line "<input>" element. Could this be addressed as part of this PR? The same warning is shown for another "<input>" in the other HTML file modified by this PR.

![image](https://github.com/Sage-Bionetworks/sage-monorepo/assets/3056480/1cf2931d-9f62-4136-a8e4-314ca2f72c7d)

Reply:
Is there any other element you found having this warning, other than the screenshot one?

Reply:
Just the two `<input>` elements mentioned. Are they also visible for you in VS Code?

Reply:
See below

Reply:
This PR is modifying two HTML files. Each of them have an `<input>` elements.

Reply:
Oh okay, but I don't see the warning on the other html file~

<img width="480" alt="Screen Shot 2024-05-16 at 12 23 13 PM" src="https://github.com/Sage-Bionetworks/sage-monorepo/assets/73901500/450553a4-f050-455c-ac11-28eb8b9db467">

Reply:
Then it's all good.